test/e2e/sagemaker/bootstrap_resources.py

```diff
@@ -20,15 +20,19 @@
 
 SAGEMAKER_SOURCE_DATA_BUCKET = "source-data-bucket-718865417152-us-west-2"
 
+"""ExecutionRoleARN needs to be created first so the role is ready to use
+before the test starts. Hence it should be the first thing to create in
+bootstrapping
+"""
 @dataclass
 class TestBootstrapResources:
-    DataBucketName: str
     ExecutionRoleARN: str
+    DataBucketName: str
```

Comment:
Both of these resources should be created before the test starts? This structure purely defines the elements of the bootstrapping resources structure, it shouldn't indicate the precedence by which they're created.